MuonAnalysis/MuonAssociators/plugins/L1MuonMatcher.cc

```diff
@@ -95,23 +107,49 @@ pat::L1MuonMatcher::produce(edm::Event & iEvent, const edm::EventSetup & iSetup)
 
     Handle<View<reco::Candidate> > reco;
     Handle<vector<l1extra::L1MuonParticle> > l1s;
-
+    Handle<l1t::MuonBxCollection> l1tBX;
+  
+    std::vector<l1t::Muon> l1ts;
+    std::vector<size_t> bxIdxs;
+
+    int minBxIdx = 0;
+    size_t l1size = 0;
+    
     iEvent.getByToken(recoToken_, reco);
-    iEvent.getByToken(l1Token_, l1s);
+
+    if (useStage2L1_) {
+      iEvent.getByToken(l1tToken_, l1tBX);
+      l1size = l1tBX->size();
+	
+      int minBX = max(firstBX_,l1tBX->getFirstBX());
+      int maxBX = min(lastBX_,l1tBX->getLastBX());
+
+      minBxIdx = l1tBX->begin(minBX) - l1tBX->begin();
+      std::copy(l1tBX->begin(minBX), l1tBX->end(maxBX), std::back_inserter(l1ts));
+
+      for (int ibx = l1tBX->getFirstBX(); ibx <= l1tBX->getLastBX(); ++ibx)
+	bxIdxs.push_back(l1tBX->end(ibx) - l1tBX->begin());
+    }
+    else {
```

Comment:
secondo me sarebbe piu' leggibile mettendo le graffe attorno al for, e mettere la graffa prima dell'else alla stessa riga dell'else, altrimenti potrebbe sembrare che la graffa a 132 chiuda il for